### PR TITLE
Map setup

### PR DESCRIPTION
**Pull request by n0remac**
When the player moves it doesn't slow down and stop. The friction must be broken somehow. I got rid of the gravity effect from the platformer tutorial so it may have something to do with that.

### CODE REVIEW

File: Core/GameInstance.py [hunk 1]
```diff
@@ -1,10 +1,92 @@
...
+            self.physics_engine.set_friction(self.game_resources.player_sprite, 0)
+        elif self.down_pressed and not self.up_pressed:
+            # Create a force to the right. Apply it.
+            force = (0, -PLAYER_MOVE_FORCE_ON_GROUND)
+            self.physics_engine.apply_force(self.game_resources.player_sprite, force)
+            # Set friction to zero for the player while moving
+            self.physics_engine.set_friction(self.game_resources.player_sprite, 0)
+        else:
+            # Player's feet are not moving. Therefore up the friction so we stop.
+            self.physics_engine.set_friction(self.game_resources.player_sprite, 1.0)
```
Comment (64-bit) [inline Core/GameInstance.py]: Is this really the correct way to stop something from moving in arcade? modifying friction seems like a really questionable way to do it,

File: run_game.py [hunk 2]
```diff
@@ -11,14 +11,46 @@ def __init__(self, width, height, title):
...
+    def on_update(self, delta_time):
+        """ Movement and game logic """
+        self.game_instance.on_update(delta_time)
+
+    def on_draw(self):
+        """ Draw everything """
+        self.game_instance.on_draw()
+
+    def on_draw_game(self):
+        self.game_instance.on_draw_game()
```
Comment (64-bit) [inline run_game.py:52]: on_draw_game is a callback for my rendering system and not arcade, and can be moved I think, entirelt inside game_instance

**Review by 64-bit — CHANGES_REQUESTED** [review]
Looks mostly good, one method I think can be removed from the game window and moved inside the game instance, and I have a question about stopping the player with friction.

File: Core/GameInstance.py (hunk 1 above)
Comment (n0remac) [reply Core/GameInstance.py]: Perhaps that was more applicable for a side scroller. The simple platformer did it with a change_x variable

`    def on_key_release(self, key, modifiers):

        if key == arcade.key.UP or key == arcade.key.W:
            self.player_sprite.change_y = 0
        elif key == arcade.key.DOWN or key == arcade.key.S:
            self.player_sprite.change_y = 0
        elif key == arcade.key.LEFT or key == arcade.key.A:
            self.player_sprite.change_x = 0
        elif key == arcade.key.RIGHT or key == arcade.key.D:
            self.player_sprite.change_x = 0`

**Review by 64-bit — APPROVED** [review]

**Pull request merged**